ssl/ssltest.c

```diff
@@ -1164,6 +1190,11 @@ int main(int argc, char *argv[])
             no_protocol = 1;
 #endif
             tls1 = 1;
+        } else if (strcmp(*argv, "-tls12") == 0) {
+#ifdef OPENSSL_NO_TLS1
```

Comment:
shouldn't that be `OPENSSL_NO_TLS1_2`?

Reply:
This targets the 1.0.2 branch, where OPENSLS_NO_TLS1_2 doesn't exist.

Reply:
Ahh...right. In which case shouldn't this be removed altogether?

Reply:
I don't think so -- OPENSSL_NO_TLS1 means no TLS 1.0, and no TLS 1.2 (and no TLS 1.1, not that this program exercises that).

Reply:
Confusing! Fair enough.